harris/harris.go

```diff
@@ -0,0 +1,195 @@
+package harris
+
+import (
+	"yi/matrix"
+	"image"
+	"yi/point"
+	"log"
+)
+const k = 0.04
+
+func smoothFilter() matrix.Matrix {
+	result := matrix.Ones(3, 3).Scale(float64(1)/float64(9))
```

Comment:
do you mean float(1)/float64(9) ? I need at least one of the number to be cast, otherwise 1/9 = 0 and will cause error output. But float64(1)/9 should work